basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/HostTestCase.java

```diff
@@ -45,7 +50,16 @@
     private static final String HOST_TBR = "host-btr_" + RandomStringUtils.randomAlphanumeric(5);
     private static final String HOST_TBA = "host-tba_" + RandomStringUtils.randomAlphanumeric(5);
 
-    private static final Address HTTP_SERVER_ADDRESS = UNDERTOW_ADDRESS.and("server", HTTP_SERVER);
+    private static final String REWRITE_FILTER_NAME = "RewriteFilterReferenceREMOVE_" + RandomStringUtils.randomAlphanumeric(5);
+    private static final Address REWRITE_FILTER_ADDRESS = UndertowConstants.UNDERTOW_FILTERS_ADDRESS.and("rewrite", REWRITE_FILTER_NAME);
+    private static final String ERROR_PAGE_FILTER_NAME = "ErrorFilterReference_" + RandomStringUtils.randomAlphanumeric(5);
+    private static final Address ERROR_PAGE_FILTER_ADDRESS = UndertowConstants.UNDERTOW_FILTERS_ADDRESS.and("error-page", ERROR_PAGE_FILTER_NAME);
+    private static final String GZIP_PAGE_FILTER_NAME = "GzipFilterReferenceADD_" + RandomStringUtils.randomAlphanumeric(5);
+    private static final Address GZIP_PAGE_FILTER_ADDRESS = UndertowConstants.UNDERTOW_FILTERS_ADDRESS.and("gzip", GZIP_PAGE_FILTER_NAME);
+    private static final String REQUEST_LIMIT_FILTER_NAME = "ReqLimitFilterReference_" + RandomStringUtils.randomAlphanumeric(5);
+    private static final Address REQUEST_LIMIT_FILTER_ADDRESS = UndertowConstants.UNDERTOW_FILTERS_ADDRESS.and("request-limit", REQUEST_LIMIT_FILTER_NAME);
```

Comment:
There is released Creaper snapshot version 1.5.1-SNAPSHOT in https://repository.jboss.org/nexus/content/repositories/snapshots/org/wildfly/extras/creaper/ containing this functionality.